manim/mobject/mobject.py

```diff
@@ -1979,13 +1979,13 @@ def restore(self) -> Self:
         self.become(self.saved_state)
         return self
 
-    def reduce_across_dimension(self, reduce_func: Callable, dim: int):
+    def reduce_across_dimension(self, reduce_func: Callable, dim: int) -> float | None:
         """Find the min or max value from a dimension across all points in this and submobjects."""
```

Comment:
As this method can now also return `None`, it would be nice to document this new behavior in the docstring. Something along the lines of:
```suggestion
    def reduce_across_dimension(self, reduce_func: Callable, dim: int) -> float | None:
        """Find the min or max value from a dimension across all points in this Mobject and its
        submobjects. This allows for using :meth:`~.length_over_dim` to calculate its length over
        a dimension, i.e. its height, width or depth. If this Mobject is empty, return ``None``,
        since this Mobject should not be taken into account when calculating lengths.
        
        Parameters
        ----------
        reduce_func
        	The reducer function to use in order to calculate a value over a dimension.
        dim
            The dimension to use. It should be 0, 1 or 2, representing the X, Y or Z coordinate,
            respectively.
            
        Returns
        -------
        float | None
            The min or max value over the dimension specified by ``dim``, or ``None`` if this
            Mobject is empty.     	
        """
```